Gruntfile.js

```diff
@@ -48,9 +48,7 @@ module.exports = function(grunt) {
           'test/lib/unittests-bundle.js': [ './test/unittests.js' ]
         },
         options: {
-          browserifyOptions: {
-            external: [ 'openpgp', 'crypto', 'node-localstorage']
-          }
+          external: [ 'openpgp']
```

Comment:
do you mean why i removed them here (you said added)?

i was about to tell you, i removed them as they are included as external by 'openpgp'; am i wrong?